templates/policy-controller.yaml

```diff
@@ -216,7 +216,7 @@ metadata:
 
 # This manifest creates a Pod Disruption Budget for Controller to allow K8s Cluster Autoscaler to evict
 
-apiVersion: policy/v1beta1
+apiVersion: policy/v1
```

Comment:
Can we include a `# CK edit` comment in here? It helps us keep track of what changes we might need to make when updating Calico and pulling a new template from upstream. See comments at top of manifest.

```suggestion
# CK edit: PodDisruptionBudget policy/v1beta1 is no longer served in k8s 1.25. Use policy/v1 instead, available since k8s 1.21
apiVersion: policy/v1
```